test/format/CONFIG.endian

```diff
@@ -4,3 +4,4 @@ logging.archive=0
 logging=1
 runs.timer=4
 runs.rows=1000000
+runs.type=row-store
```

Comment:
This is another change that should be reverted if we ever make column-store work again.

Reply:
I'll create a ticket to revert the change once we have column-store working with `format`.